src/ddoc_cache/src/ddoc_cache_lru.erl

```diff
@@ -143,11 +146,11 @@ handle_call(Msg, _From, St) ->
 
 
 handle_cast({evict, DbName}, St) ->
-    gen_server:abcast(mem3:nodes(), ?MODULE, {do_evict, DbName}),
+    gen_server:abcast(mem3:nodes(), ?OPENER, {do_evict, DbName}),
     {noreply, St};
 
 handle_cast({refresh, DbName, DDocIds}, St) ->
-    gen_server:abcast(mem3:nodes(), ?MODULE, {do_refresh, DbName, DDocIds}),
+    gen_server:abcast(mem3:nodes(), ?OPENER, {do_evict, DbName, DDocIds}),
```

Comment:
Is changing the destination module and the cast message to `do_evict` intentional?

Reply:
Yes, that's the fix.

https://github.com/apache/couchdb/blob/master/src/ddoc_cache/src/ddoc_cache_opener.erl#L47-L55

Its reverting the message passing protcol to match the old version. For old nodes this means they continue to behave the same. For new nodes, the ddoc_cache_opener module exists solely to forward and translate the old style messages to new style. My mistake was just not realizing that ddoc_cache_lru wouldn't be silently ignored by the old version (because the ets_lru process exists).

Reply:
I should note that this is a bit convoluted because of the upgrade. In the future after the new ddoc_cache has been on master for enough versions we'll remove ddoc_cache opener and revert this commit to avoid the jump through ddoc_cache_opener.

As a note for the intrepid observer, upgrades that involve changing messages passed between nodes are totes the hardest.

Reply:
aha! Makes sense now.